src/modules/planetside/CensusApi.ts

```diff
@@ -141,6 +141,23 @@ class CensusApi {
     }
   }
 
+  async getCharacter(query: QueryObject<Character>) {
+    type Item = Character & {
```

Comment:
Please add the cert data to the Character type in types.ts, so you don't need any custom Item ere.